experimental/README.md

```diff
@@ -0,0 +1,8 @@
+# Experimental
+
+This directory is intended to contain files that should be version-controlled,
+are experimental in nature, and do not have (long term) dependencies from
+production.
+
+In particular, files associated with experiments running in production do not
+belong here.
```

Comment:
Borrowing the idea from a former employee. My intent is to submit the PR establishing the directory, merge staging, and eliminate the directory and readme from this PR.